test.sh

```diff
@@ -4,10 +4,15 @@ readonly compose_file='docker-compose.test.yml'
 readonly v5_puppet_gem="~> 5.5.21"
```

Comment:
This should match the Gemfile. I'm guessing we should probably just remove this line so that we use the default from gemfile.

Reply:
updated.